src/amo/components/AddonAuthorLinks/index.js

```diff
@@ -0,0 +1,72 @@
+/* @flow */
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import translate from 'core/i18n/translate';
+import type { AddonType } from 'core/types/addons';
+import DefinitionList, { Definition } from 'ui/components/DefinitionList';
+import type { I18nType } from 'core/types/i18n';
+import type { AppState } from 'amo/store';
+import { isAddonAuthor } from 'core/utils';
+
+type Props = {|
+  addon: AddonType | null,
+|};
+
+type InternalProps = {|
+  ...Props,
+  i18n: I18nType,
+  userId: number | null,
+|};
+
+export class AddonAuthorLinksBase extends React.Component<InternalProps> {
+  render() {
+    const { addon, i18n, userId } = this.props;
+
+    if (addon === null) {
+      return null;
+    }
+
+    const isAuthor = isAddonAuthor({ addon, userId });
```

Comment:
I think, for now, we shouldn't display anything if the user is not the author of the add-on. That is because otherwise the list of links will be empty for many users (even anonymous users). Example:

<img width="485" alt="Screen Shot 2020-03-23 at 17 35 06" src="https://user-images.githubusercontent.com/217628/77339852-b459e400-6d2c-11ea-89bf-d1a29cb05fc8.png">